include/net/net_if.h

```diff
@@ -78,8 +78,6 @@ struct net_if_addr {
 
 	/** Is this IP address usage limited to the subnet (mesh) or not */
 	uint8_t is_mesh_local : 1;
-
-	uint8_t _unused : 5;
```

Comment:
It is not unnecessary. It makes it obvious that if one want to add another field, there is that amount of space left. We do that everywhere in zephyr. It's totally fine to keep it.

Reply:
> It is not unnecessary. It makes it obvious that if one want to add another field, there is that amount of space left. We do that everywhere in zephyr. It's totally fine to keep it.

IMHO this field is quite useless actually, it does not really give that much information. But we can leave it if needed.